FRPs/FRP-6.md

```diff
@@ -0,0 +1,53 @@
+# FRP-6: Literature review of auction mechanism that fit in with Flashbots.
+
+
+**MEV Fellow:** @sbaks0820
+</br> **Status:** Draft
+</br> **Updated:** 24.11.2020
+
+### Abstract
+The puspose of this isssue is to investigate and survey the existing literature of auction mechanism design to motivate the design of the block space and transaction ordering auctions enabled by MEV-GETH.
+An MEV-GETH auction is different from a traditional auction in that users submit and bid on tx sequences that are, potentially, non-overlapping.
+The current implementation of MEV-GETH accepts only a single transacton sequence into a block, however, a more sophisticated auction mechanism might permit multiple "bundles" with interaction between winning, non-overlapping bundles.
+Therefore careful consideration is required to design a good auction mechanism for MEV-GETH.
+In addition to surveying existing literature, it is impotant to survey auction designs in existing crypto projects so understand the practical requirements of a decentralized auction such as privacy, security, auditability, and efficiency.
+
+### Defining the Research Question
+
+The high-level research question is: What is a good auction mechanism for MEV-GETH auctions? 
+
+The first step in answering this broad question is first surveying and evaluating the existing literature and practice of auction mechanism design in a decentralized setting.
+In order to tackle this objects is to answer the following questions:
+* In what ways do crypto auctions differ from existing literature: what are the impact of easy collusion forming on cooperative strategies and does current literature fall short of addressing this concern? Ad auctions have a similar setting, perhaps a better survey of how they deal with it? (see Methodolody for survey of existing projects, i.e. industry anecdotes). 
+* What assumptions can be made about the participants and their capabilities (includes a potentially malicious miner)?
+* Does expanding the auction format (from a single winning bundle to potentially multipe winning bundles) make auction design much more difficult?
+* How do current implementation auctions in crypto deal with issues such as efficiency, security, privacy, and auditability?
+* In the field of combinatorial auctions: are there existing theoretical auctions that imply constraints on our setting in terms of computability of optimal strategy?
+* In the area of computational complexity: are there computationally difficult auctions with good enough heuristic, or sub-optimal, solutions?
```

Comment:
This gets back to my point of how you want to define "approximate" --- you'll get very different outcomes for _iterated_ auctions, unfortunately!